source/reference/operator/query/mod.txt

```diff
@@ -117,3 +117,31 @@ operator with an array that contains four elements:
    	"$err" : "bad query: BadValue malformed mod, too many elements",
    	"code" : 16810
    }
+
+Modulo with Decimal Divisors
```

Comment:
In the previous review, Nikita suggested changing this section title to "Floating Point Arguments". Did we want to incorporate that suggestion?